ZenPacks/zenoss/Dashboard/browser/resources/js/defaultportlets.js

```diff
@@ -359,7 +359,11 @@
             this.callParent(arguments);
         },
         getIFrameSource: function() {
-            return Ext.String.format('{0}/simpleLocationGeoMap?polling={1}', Zenoss.render.link(null, this.baselocation), this.pollingrate);
+            var location = Zenoss.render.link(null, this.baselocation);
+            if (location.indexOf('/zport/dmd/Locations') < 0) {
+                return null;
```

Comment:
return empty location to not refresh map every time when user type wrong location.